shippo/test/helper.py

```diff
@@ -423,24 +424,21 @@ def setUp(self):
         self._shippo_original_attributes = {}
 
         for attr in self.RESTORE_ATTRIBUTES:
-            self._shippo_original_attributes[attr] = getattr(
-                shippo.config, attr)
-
+            self._shippo_original_attributes[attr] = getattr(config, attr)
         api_base = os.environ.get('SHIPPO_API_BASE')
         if api_base:
-            shippo.config.api_base = api_base
+            config.api_base = api_base
 
-        shippo.config.api_key = os.environ.get(
-            'SHIPPO_API_KEY', '51895b669caa45038110fd4074e61e0d')
-        shippo.config.api_version = os.environ.get(
-            'SHIPPO_API_VERSION', '2018-02-08')
+        config.api_key = os.environ.get('SHIPPO_API_KEY', '51895b669caa45038110fd4074e61e0d')
+        config.api_version = os.environ.get('SHIPPO_API_VERSION', '2018-02-08')
+        config.app_name = os.environ.get('SHIPPO_API_KEY', 'MyAwesomeApp')
```

Comment:
Is this correct? config.app_name = os.environ.get('SHIPPO_API_KEY', 'MyAwesomeApp')

Reply:
Nice catch. It should be `APP_NAME` here. Thanks

Reply:
os.environ.get('SHIPPO_API_KEY', 'MyAwesomeApp') --- Still see Shippo_API_KEY here

Reply:
It is an outdated version of that file. Can you please check if this one is fine?
https://github.com/goshippo/shippo-python-client/pull/87/files#diff-b9d835d53f2525e907d924e6a2fb82794fbca00e3a4702988dbf75d97f11feafR434